packages/subscription-manager/src/components/BlockEmailsSetting/style.scss

```diff
@@ -0,0 +1,26 @@
+.block-emails-setting {
+	display: flex;
+	flex-direction: column;
+
+	&.form-fieldset {
+		margin-left: 2px;
+	}
+
+	.title {
+		margin-bottom: 7px;
+		color: #50575e;
```

Comment:
We should stick to using the studio constants where possible: `$studio-gray-60` like [here](https://github.com/Automattic/wp-calypso/blob/c613d29ce84c6fbd38221884af1b9000a01de5c1/packages/subscription-manager/src/components/TabsSwitcher/styles.scss#L35).

Reply:
Cool. I'll change it 👍